### ISSUE TITLE
Fails to convert blocks where an image has no alt text

### ISSUE THREAD

**Issue by ForgottenHero** [10up/convert-to-blocks#130]
### Describe the bug

In my classic text I have many images throughout but all are lacking alt text.
After importing my content and enabling this plugin, any paragraph that includes an image displays an error instead.
**Original:**
`<img class="alignnone size-full wp-image-40771" src="https://mywebsiteaddress.com/wp-content/uploads/2023/04/A3_Incomplete_6.png" alt="" width="687" height="484" />`
**After conversion:**
`<img class="alignnone size-full wp-image-40771" src="https://mywebsiteaddress.com/wp-content/uploads/2023/04/A3_Incomplete_6.png" alt=" width=" 687"="" height="484">`

### Steps to Reproduce

1. Create a post using the classic editor including text and an image where there is no "alt=" value.
2. Enable the Convert to Blocks plugin
3. Refresh the article
4. See the error: "This block has encountered an error and cannot be previewed." where the image should've been.

### Screenshots, screen recording, code snippet

![image](https://github.com/10up/convert-to-blocks/assets/4233945/2d26d654-b318-4e3a-80bb-cfed3e5643d8)
![image](https://github.com/10up/convert-to-blocks/assets/4233945/2bdd45f3-8d0b-4c39-a277-6f916352d6c6)

Original text:
`<img class="alignnone size-full wp-image-40771" src="https://supportportal.covetrus.com/wp-content/uploads/2019/08/A3_Incomplete_6.png" alt="" width="687" height="484" />`

Converted text:
`<img class="alignnone size-full wp-image-40771" src="https://attractive-fowl.flywheelsites.com/wp-content/uploads/2023/04/A3_Incomplete_6.png" alt=" width=" 687"="" height="484">`

### Environment information

- Device: Windows PC
- OS: Windows 10 Version 22H2 Build 19045.3208
- Browser and version: Chrome Version 115.0.5790.102 (Official Build) (64-bit)

### WordPress information

`
### wp-core ###

version: 6.2.2
site_language: en_AU
user_language: en_AU
timezone: Australia/Brisbane
permalink: /%category%/%postname%/
https_status: true
multisite: false
user_registration: 0
blog_public: 1
default_comment_status: undefined
environment_type: production
user_count: 13
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /wordpress
wordpress_size: loading...
uploads_path: /www/wp-content/uploads
uploads_size: loading...
themes_path: /www/wp-content/themes
themes_size: loading...
plugins_path: /www/wp-content/plugins
plugins_size: loading...
database_size: loading...
total_size: loading...

### wp-dropins (1) ###

db-error.php: true

### wp-active-theme ###

name: Hello Elementor (hello-elementor)
version: 2.7.1 (latest version: 2.8.1)
author: Elementor Team
author_website: https://elementor.com/?utm_source=wp-themes&utm_campaign=author-uri&utm_medium=wp-dash
parent_theme: none
theme_features: core-block-patterns, widgets-block-editor, menus, post-thumbnails, automatic-feed-links, title-tag, html5, custom-logo, editor-style, align-wide, woocommerce, wc-product-gallery-zoom, wc-product-gallery-lightbox, wc-product-gallery-slider, widgets
theme_path: /www/wp-content/themes/hello-elementor
auto_update: Disabled

### wp-themes-inactive (1) ###

Twenty Twenty-Three: version: 1.1, author: the WordPress team, Auto-updates disabled

### wp-plugins-active (37) ###

BetterDocs: version: 2.3.7, author: WPDeveloper, Auto-updates disabled
BetterDocs Pro: version: 2.2.7, author: WPDeveloper, Auto-updates disabled
Convert to Blocks: version: 1.2.0, author: 10up, Auto-updates disabled
Elementor: version: 3.14.0, author: Elementor.com (latest version: 3.14.1), Auto-updates disabled
Elementor Pro: version: 3.14.0, author: Elementor.com (latest version: 3.14.1), Auto-updates disabled
Essential Addons for Elementor: version: 5.8.0, author: WPDeveloper (latest version: 5.8.3), Auto-updates disabled
Essential Addons for Elementor - Pro: version: 5.4.11, author: WPDeveloper (latest version: 5.5.0), Auto-updates disabled
Essential Blocks: version: 4.1.2, author: WPDeveloper (latest version: 4.1.5), Auto-updates disabled
Favorites: version: 2.3.3, author: Kyle Phillips, Auto-updates disabled
IdeaPush - Feature Request Manager: version: 8.46, author: Martin Gibson, Auto-updates disabled
If Menu - Visibility control for menus: version: 0.17.0, author: Layered, Auto-updates disabled
Mammoth .docx converter: version: 1.20.0, author: Michael Williamson, Auto-updates disabled
Managed Plugin Updates: version: 5.17.7, author: Flywheel, Auto-updates disabled
MemberPress Account Nav Tabs: version: 1.0.0, author: Caseproof, LLC, Auto-updates disabled
MemberPress Courses: version: 1.2.10, author: Caseproof LLC (latest version: 1.2.11), Auto-updates disabled
MemberPress Elementor Content Protection: version: 1.0.6, author: Caseproof, LLC, Auto-updates disabled
MemberPress Importer: version: 1.6.15, author: Caseproof, LLC, Auto-updates disabled
MemberPress Pro: version: 1.11.6, author: Caseproof, LLC (latest version: 1.11.12), Auto-updates enabled
Members: version: 3.2.5, author: MemberPress, Auto-updates disabled
OoohBoi Steroids for Elementor: version: 2.1.5, author: OoohBoi, Auto-updates disabled
OTGS Installer: version: 3.0.3, author: OnTheGoSystems, Auto-updates disabled
Rank Math SEO: version: 1.0.117.1, author: Rank Math (latest version: 1.0.119.1), Auto-updates disabled
Rank Math SEO PRO: version: 3.0.38, author: Rank Math (latest version: 3.0.40), Auto-updates disabled
Relevanssi Premium: version: 2.23.0, author: Mikko Saari, Auto-updates disabled
Remove Dashboard Access: version: 1.1.5, author: TrustedLogin, Auto-updates disabled
Simple Feature Requests: version: 2.2.5.1, author: Mindsize, Auto-updates disabled
The Plus Addons for Block Editor: version: 3.0.1, author: POSIMYTH (latest version: 3.0.4), Auto-updates disabled
The Plus Addons for Elementor: version: 5.2.8, author: POSIMYTH (latest version: 5.2.12), Auto-updates disabled
Toolset Access: version: 2.8.16, author: OnTheGoSystems, Auto-updates disabled
Toolset Blocks: version: 1.6.7, author: OnTheGoSystems, Auto-updates disabled
Toolset Forms: version: 2.6.16, author: OnTheGoSystems, Auto-updates disabled
Toolset Types: version: 3.4.19, author: OnTheGoSystems, Auto-updates disabled
User Switching: version: 1.7.0, author: John Blackbourn & contributors, Auto-updates disabled
WP All Export Pro: version: 1.8.3, author: Soflyy, Auto-updates disabled
WP All Import - Toolset Types Add-On Pro: version: 1.0.6, author: Soflyy, Auto-updates disabled
WP All Import - User Import Add-On Pro: version: 1.1.8, author: Soflyy, Auto-updates disabled
WP All Import Pro: version: 4.8.0, author: Soflyy, Auto-updates disabled

### wp-plugins-inactive (2) ###

MemberPress Corporate Accounts: version: 1.5.31, author: Caseproof, LLC, Auto-updates disabled
Sucuri Security - Auditing, Malware Scanner and Hardening: version: 1.8.39, author: Sucuri Inc., Auto-updates disabled

### wp-media ###

image_editor: WP_Image_Editor_Imagick
imagick_module_version: 1690
imagemagick_version: ImageMagick 6.9.10-23 Q16 x86_64 20190101 https://imagemagick.org
imagick_version: 3.7.0
file_uploads: File uploads is turned off
post_max_size: 300M
upload_max_filesize: 300M
max_effective_size: 300 MB
max_file_uploads: 20
imagick_limits: 
	imagick::RESOURCETYPE_AREA: 51 GB
	imagick::RESOURCETYPE_DISK: 1.844674407371E+19
	imagick::RESOURCETYPE_FILE: 786432
	imagick::RESOURCETYPE_MAP: 51 GB
	imagick::RESOURCETYPE_MEMORY: 25 GB
	imagick::RESOURCETYPE_THREAD: 1
	imagick::RESOURCETYPE_TIME: 1.844674407371E+19
imagemagick_file_formats: 3FR, 3G2, 3GP, AAI, AI, ART, ARW, AVI, AVS, BGR, BGRA, BGRO, BIE, BMP, BMP2, BMP3, BRF, CAL, CALS, CANVAS, CAPTION, CIN, CIP, CLIP, CMYK, CMYKA, CR2, CRW, CUR, CUT, DATA, DCM, DCR, DCX, DDS, DFONT, DNG, DPX, DXT1, DXT5, EPDF, EPI, EPS, EPS2, EPS3, EPSF, EPSI, EPT, EPT2, EPT3, ERF, FAX, FILE, FITS, FRACTAL, FTP, FTS, G3, G4, GIF, GIF87, GRADIENT, GRAY, GRAYA, GROUP4, H, HALD, HDR, HISTOGRAM, HRZ, HTM, HTML, HTTP, HTTPS, ICB, ICO, ICON, IIQ, INFO, INLINE, IPL, ISOBRL, ISOBRL6, JBG, JBIG, JNG, JNX, JPE, JPEG, JPG, JPS, JSON, K25, KDC, LABEL, M2V, M4V, MAC, MAGICK, MAP, MASK, MAT, MATTE, MEF, MIFF, MKV, MNG, MONO, MOV, MP4, MPC, MPEG, MPG, MRW, MSL, MTV, MVG, NEF, NRW, NULL, ORF, OTB, OTF, PAL, PALM, PAM, PATTERN, PBM, PCD, PCDS, PCL, PCT, PCX, PDB, PDF, PDFA, PEF, PES, PFA, PFB, PFM, PGM, PGX, PICON, PICT, PIX, PJPEG, PLASMA, PNG, PNG00, PNG24, PNG32, PNG48, PNG64, PNG8, PNM, PPM, PREVIEW, PS, PS2, PS3, PSB, PSD, PTIF, PWP, RADIAL-GRADIENT, RAF, RAS, RAW, RGB, RGBA, RGBO, RGF, RLA, RLE, RMF, RW2, SCR, SCT, SFW, SGI, SHTML, SIX, SIXEL, SPARSE-COLOR, SR2, SRF, STEGANO, SUN, TEXT, TGA, THUMBNAIL, TIFF, TIFF64, TILE, TIM, TTC, TTF, TXT, UBRL, UBRL6, UIL, UYVY, VDA, VICAR, VID, VIFF, VIPS, VST, WBMP, WEBP, WMV, WPG, X, X3F, XBM, XC, XCF, XPM, XPS, XV, XWD, YCbCr, YCbCrA, YUV
gd_version: 2.3.3
gd_formats: GIF, JPEG, PNG, WebP, BMP, AVIF, XPM
ghostscript_version: unknown

### wp-server ###

server_architecture: Linux 5.15.89+ x86_64
httpd_software: Flywheel/5.1.0
php_version: 8.1.20 64bit
php_sapi: fpm-fcgi
max_input_variables: 10000
time_limit: 178
memory_limit: 200M
max_input_time: 60
upload_max_filesize: 300M
php_post_max_size: 300M
curl_version: 7.68.0 OpenSSL/1.1.1f
suhosin: false
imagick_availability: true
pretty_permalinks: true
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 5.7.42-google
client_version: mysqlnd 8.1.20
max_allowed_packet: 33554432
max_connections: 4030

### wp-constants ###

WP_HOME: https://attractive-fowl.flywheelsites.com
WP_SITEURL: https://attractive-fowl.flywheelsites.com
WP_CONTENT_DIR: /www/wp-content
WP_PLUGIN_DIR: /www/wp-content/plugins
WP_MEMORY_LIMIT: 200M
WP_MAX_MEMORY_LIMIT: 200M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_ENVIRONMENT_TYPE: Undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: not writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable
mu-plugins: writable

`

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by kmgalanakis** [comment]
@ForgottenHero thank you for reporting this issue.

Unfortunately, I'm unable to reproduce this. When I enable the plugin and refresh the post edit page, the post content is automatically conv##erted to blocks.

### Original content

```
Lorem ipsum

<img class="alignnone size-full wp-image-40771" src="https://cdn.pixabay.com/photo/2016/09/07/16/19/pile-1651945_1280.jpg" alt="" width="687" height="484" />
```

### Converted content

```
<!-- wp:paragraph -->
<p>Lorem ipsum</p>
<!-- /wp:paragraph -->

<!-- wp:image {"id":40771} -->
<figure class="wp-block-image"><img src="https://cdn.pixabay.com/photo/2016/09/07/16/19/pile-1651945_1280.jpg" alt="" class="wp-image-40771"/></figure>
<!-- /wp:image -->
```

Am I missing something?

**Comment by ForgottenHero** [comment]
Sorry for the delayed reply.
I removed all alt-text from images and imported a fresh batch of content into the site.

The images and content is displaying without issue now.
I don't know what exactly was going wrong, but I'd assume it's a unique edge-case or a non-issue.
